inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/config.cpp

```diff
@@ -12,11 +12,18 @@ namespace {
             InferenceEngine::Precision::FP16
     };
 
-    const std::vector<std::map<std::string, std::string>> conf = {
+    const std::vector<std::map<std::string, std::string>> config = {
             {}
     };
 
-    const std::vector<std::map<std::string, std::string>> Configs = {
+    INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests, CorrectConfigAPITests,
+                             ::testing::Combine(
+                                     ::testing::ValuesIn(netPrecisions),
+                                     ::testing::Values(CommonTestUtils::DEVICE_CPU),
```

Comment:
why do we need add CPU test case in this PR?

Reply:
not added, just modified code location, will revert these changes